From f0634763aefa77f80d34f555de9bf3d8a40ddc50 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Fri, 10 Nov 2017 16:19:58 -0500
Subject: [PATCH] NET-381 use triton-origin for booter/dhcpd TRITON-13 service
 configure.sh scripts are of inconsistent strictness NET-380 dhcpd service is
 only accidenlty enabled Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 Makefile          |  4 ++--
 boot/configure.sh | 13 ++++++-------
 2 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index dafc197..84cd9de 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 NAME=dhcpd
@@ -47,7 +47,7 @@ JSSTYLE_FLAGS = -o indent=4,doxygen,unparenthesized-return=0
 NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
-	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
 
 
diff --git a/boot/configure.sh b/boot/configure.sh
index bfae9c7..783ff9c 100755
--- a/boot/configure.sh
+++ b/boot/configure.sh
@@ -7,18 +7,17 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+set -o errexit
+set -o pipefail
 set -o xtrace
 
-echo "Importing dhcpd manifest"
+echo "Importing dhcpd manifest (default: enabled)"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/dhcpd.xml
 
-echo "Enabling dhcpd service"
-/usr/sbin/svcadm enable smartdc/site/dhcpd
-
 echo "Importing tftpd manifest"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/tftpd.xml
 
@@ -29,9 +28,9 @@ echo "Configuring nginx"
 cp /opt/smartdc/booter/etc/nginx.conf /opt/local/etc/nginx/nginx.conf
 
 echo "Importing nginx manifest"
-/usr/sbin/svccfg import /opt/local/share/smf/nginx/manifest.xml
+/usr/sbin/svccfg import /opt/local/lib/svc/manifest/nginx.xml
 
 echo "Enabling nginx service"
-/usr/sbin/svcadm enable network/nginx
+/usr/sbin/svcadm enable pkgsrc/nginx
 
 exit 0
-- 
2.21.0

